src/app/SharpRaven/Data/Breadcrumb.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+
+using SharpRaven.Serialization;
+
+namespace SharpRaven.Data {
+    /// <summary>
+    /// Breadcrumb trail.
+    /// </summary>
+    public class Breadcrumb {
+        private readonly DateTime timestamp;
+
+        public Breadcrumb(string category) 
+        {
+            Category = category;
```

Comment:
I tink should be similar the Log frameworks (like Log4*), where an exception does not affect the functionality of the application. So i guess the put the default category. Do you agree?